multiqc/modules/multivcfanalyzer/multivcfanalyzer.py

```diff
@@ -0,0 +1,170 @@
+
+        # Find and load any MultiVCFAnalyzer reports
+        self.mvcf_data = dict()
+
+        # Find and load JSON file
+        for f in self.find_log_files('multivcfanalyzer', filehandles=True):
+            self.parse_data(f)
+
+        # Filter samples
+        self.mvcf_data = self.ignore_samples(self.mvcf_data)
+
+        # Return if no samples found
+        if len(self.mvcf_data) == 0:
+            raise UserWarning
+
+        # Add in extra columns to data file
+        self.compute_perc_hets()
+
+        # Save data output file
+        self.write_data_file(self.mvcf_data, 'multiqc_multivcfanalyzer_metrics')
+
+        # Add to General Statistics
+        self.addSummaryMetrics()
+
+
+    def parse_data(self, f):
+        try:
+            data = json.load(f['f'])
+        except Exception as e:
+            log.debug(e)
+            log.warn("Could not parse MultiVCFAnalyzer JSON: '{}'".format(f['fn']))
+            return
+
+        # Parse JSON data to a dict
+        for s_name in data:
+            if (s_name == 'metadata'):
+                continue
+            
+            if (s_name == 'metrics'):
+                for sample in data['metrics'].items():
+                    s_clean = sample[0]
+                    if s_clean in self.mvcf_data:
+                        log.debug("Duplicate sample name found! Overwriting: {}".format(s_clean))
+
+                    self.add_data_source(f, s_clean)
+                    self.mvcf_data[s_clean] = dict()
+
+                    for k, v in sample[1].items():
+                        try:
+                            self.mvcf_data[s_clean][k] = float(v)
+                        except ValueError:
+                            self.mvcf_data[s_clean][k] = v
+
+    #Compute % heterozygous snp alleles and add to data
+    def compute_perc_hets(self):
+        """Take the parsed stats from MultiVCFAnalyzer and add one column per sample """
+        for sample in self.mvcf_data:
+            self.mvcf_data[sample]['Heterozygous SNP alleles (percent)'] = (self.mvcf_data[sample]['SNP Calls (het)'] / self.mvcf_data[sample]['SNP Calls (all)'])*100
+
+    def addSummaryMetrics(self):
+        """ Take the parsed stats from MultiVCFAnalyzer and add it to the main plot """
+
+        headers = OrderedDict()
+        headers['SNP Calls (all)'] = {
+            'title': 'All SNP calls',
+            'description': 'Total number of non-reference calls made',
+            'scale': 'OrRd',
+            'shared_key': 'snp_call'
+        }
+        headers['SNP Calls (het)'] = {
+            'title': 'Heterozygous SNP Calls',
+            'description': 'Total number of non-reference calls not passing homozygosity thresholds',
+            'scale': 'OrRd',
+            'hidden': True,
+            'shared_key': 'snp_call'
+        }
+        headers['Heterozygous SNP alleles (percent)'] = {
```

Comment:
Column titles are too long..

```suggestion
        headers['% Hets'] = {
```